src/code/z_camera_data.c

```diff
@@ -355,7 +405,7 @@ CameraModeValue sSetNormal3ModeTalkData[] = {
     CAM_FUNCDATA_KEEP3(-30, 70, 200, 40, 10, 10, 20, 70, 45, 10, 10, 0x3500),
 };
 
-/* These values are for when the eye
+/* These values are for when the eye is
  * >= OREG(45) units below the surface of the water.
  */
```

Comment:
Not part of your change but while we're looking at it:
```suggestion
/**
 * These values are for when the eye is >= OREG(45) units below the surface of the water.
 */
```